lms/templates/bookmarks/bookmarks-list.underscore

```diff
@@ -7,15 +7,17 @@
 
     <div class='bookmarks-results-list'>
         <% bookmarksCollection.each(function(bookmark, index) { %>
-             <a class="bookmarks-results-list-item" href="<%= bookmark.blockUrl() %>" aria-labelledby="bookmark-link-<%= index %>" data-bookmark-id="<%= bookmark.get('id') %>" data-component-type="<%= bookmark.get('block_type') %>" data-usage-id="<%= bookmark.get('usage_id') %>" aria-describedby="bookmark-type-<%= index %> bookmark-date-<%= index %>">
+             <a class="bookmarks-results-list-item" href="<%- bookmark.blockUrl() %>" aria-labelledby="bookmark-link-<%- index %>" data-bookmark-id="<%- bookmark.get('id') %>" data-component-type="<%- bookmark.get('block_type') %>" data-usage-id="<%- bookmark.get('usage_id') %>" aria-describedby="bookmark-type-<%- index %> bookmark-date-<%- index %>">
                 <div class="list-item-content">
                     <div class="list-item-left-section">
-                        <h3 id="bookmark-link-<%= index %>" class="list-item-breadcrumbtrail"> <%= _.map(_.pluck(bookmark.get('path'), 'display_name'), _.escape).concat([_.escape(bookmark.get('display_name'))]).join(' <i class="icon fa fa-caret-right" aria-hidden="true"></i><span class="sr">-</span> ') %> </h3>
-                        <p id="bookmark-date-<%= index %>" class="list-item-date"> <%= gettext("Bookmarked on") %> <%= humanFriendlyDate(bookmark.get('created')) %> </p>
+                        <h3 id="bookmark-link-<%- index %>" class="list-item-breadcrumbtrail">
+                            <%= bookmark.breadcrumbTrail() %>
```

Comment:
This is the scenario where you would use `ensureHtml`:
```
<%= HtmlUtils.ensureHtml(bookmark.breadcrumbTrail()) %>
```
It won't do anything different here except ensure to the linter that this is properly handled.
Note that if you aren't already using it, you can use `HtmlUtils.template()` to load this underscore template and make `HtmlUtils` automatically accessible from here.